.test-infra/jenkins/job_InferenceBenchmarkTests_Python.groovy

```diff
@@ -0,0 +1,166 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import LoadTestsBuilder as loadTestsBuilder
```

Comment:
Is it just me, or does anyone see these red-ish color blocks here?
![Screen Shot 2022-08-22 at 10 27 55 AM](https://user-images.githubusercontent.com/14128880/185945985-672ffd36-6f0a-4190-9daf-8e7f9db1073b.png)

Reply:
I see it too

Reply:
I think these come from weird space characters via copy paste? I've seen them elsewhere and they've been harmless